src/app/modules/practitioner-registration/pages/practitioner-attachment/practitioner-attachment.component.ts

```diff
@@ -101,6 +101,27 @@ export class PractitionerAttachmentComponent extends BcpBaseForm implements OnIn
     }
     this.formGroup.valueChanges.subscribe( value => {
       // Update data service values
+      switch (value.attachmentType) {
```

Comment:
Dates should be set to use:
  attachmentEffectiveDate: Date;
  attachmentCancelDate: Date;